assessment/cog-memory-leaks.robot

```diff
@@ -0,0 +1,104 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# Required setup keywords:
+# Prepare Test Suite - elementary setup keyword for all tests.
+# Upload Required Images - uploads all required files onto the PiKVM.
+# Required teardown keywords:
+# Log Out And Close Connection - elementary teardown keyword for all tests.
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keywords
+...                     Log Out And Close Connection
+# Before the tests are executed the platform must be booted into Linux and have
+# Weston running in the background. The user should be logged in as root.
+
+
+*** Test Cases ***
+COG001.001 Check memory usage on heavy application
+    [Documentation]    This test measures memory usage while running a
+    ...    memory-heavy web application in Cog browser.
+    ${variables}=    Get Variables
+    IF    "\${TIME}" not in $variables
```

Comment:
@PLangowski do we use `${TIME}` in the global scope/other files? If it's related to the test suite only, maybe we could use Suite Setup to do this once?

Reply:
Done f3a9922863cff3fa1098b1a4e7450bfd8697e7c6